Assets/PatchKit Patcher/Scripts/AppData/Remote/Downloaders/HttpDownloader.cs

```diff
@@ -145,8 +145,15 @@ public void Download(CancellationToken cancellationToken)
                     }
                 }
 
-                DebugLogger.Log("Waiting 10 seconds before trying again...");
-                Thread.Sleep(10000);
+                int retrySleepDuration = 10000;
+                int singleSleep = 100;
+                DebugLogger.LogFormat("Waiting {0} seconds before trying again...", retrySleepDuration/1000);
+                // FIX: Bug #692
```

Comment:
If possible please paste the issue title here.

Reply:
Done